ang/volunteer/VolOppsCtrl.html

```diff
@@ -16,7 +16,12 @@
   </div>
 
   <!-- Search Form -->
-  <div class="crm-vol-half-responsive crm-vol-opp-search">
+  <div class="crm-vol-half-responsive crm-vol-opp-search" ng-show="allowShowSearch">
```

Comment:
The wrapper for the button has the same class name as the wrapper for search form. Copy/paste error?